### PR TITLE
Add spinner while sending bytes to the device

### LINKED ISSUE #49: Improve UX for sending bytes
### Description
currently User clicks send , No visual representation is shown
button becomes non-responsive

### ScreenShots
Non Responsive UX when sendBytes() works

### Expected UX
1) Details(Or Name) of device where bytes are being sent should be shown to user
2) Progress when bytes are being sent to user
3) failure message when sendBytes() fails
4) `Send` button should be disabled when bluetooth found

### Would you like to work
 YES

### PR DESCRIPTION
**Pull request by yatri1609**
Fixes #49 

Changes:
Added spinner while sending bits to the device.

Screenshots for the change:

### CODE REVIEW

**Comment by yatri1609** [comment]
@iamareebjamal @adityastic @yashk2000 Please review this PR

**Comment by yashk2000** [comment]
I'm not sure whether this is needed or not. Instead of the spinner, disabling the send button will also work and is a better and shorter way to tell the user that send has been pressed. Again, these are my views, I'm not sure.

@iamareebjamal could you please clarify this?

**Comment by yatri1609** [comment]
@yashk2000 I think this works more efficiently in context of UX as user will have a better experience with this.

**Comment by yatri1609** [comment]
@CloudyPadmal can you also please review this ?

**Comment by iamareebjamal** [comment]
There are conflicts and the build is failing

**Comment by adityastic** [comment]
Hey, @yatri1609 @yashk2000, both of your PR fix the same issue of giving the feedback about the bytes being sent. I'm not sure what to do, as @yashk2000 's request got merged already

**Comment by yatri1609** [comment]
@adityastic But this has better User Experience while sending bits so I think this is better one. And @iamareebjamal I'll fix the conflicts

**Comment by adityastic** [comment]
> @adityastic But this has better User Experience while sending bits so I think this is better one. And also merging of one kind of solution doesn't stop the app from improving. And @iamareebjamal I'll fix the conflicts

Alright, Let me review it for you

File: app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt [hunk 1]
```diff
@@ -15,13 +15,9 @@ import android.support.v4.content.ContextCompat
...
 import android.support.v7.widget.RecyclerView
+import android.view.View
 import android.view.inputmethod.InputMethodManager
-import android.widget.Button
-import android.widget.CheckBox
-import android.widget.EditText
-import android.widget.Spinner
-import android.widget.Toast
-import android.widget.ArrayAdapter
+import android.widget.*
```
Comment (adityastic) [inline app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt]: avoid using wildcard imports

Comment (adityastic) [reply app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt]: they usually break the build in travis

Comment (yatri1609) [reply app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt]: Yeah I got to fix it !

**Comment by yatri1609** [comment]
@adityastic Please have a look now

**Comment by adityastic** [comment]
@yatri1609 great

**Pull request merged**